src/controllers/oauth_controller.ts

```diff
@@ -175,8 +175,8 @@ function verifyClientAuthorization(client: ClientRegistry,
     return false;
   }
 
-  let clientId = decodeURIComponent(parts[0]);
-  let clientSecret = decodeURIComponent(parts[1]);
+  let clientId = decodeURIComponent(parts[0].replace(/\+/g, '%20'));
```

Comment:
What behavior does this fix/prevent?

Reply:
The simple-oauth2 library changed the encoding of URL parameters. This was added to handle that.